app/manifest.json

```diff
@@ -55,8 +55,13 @@
     },
     {
       "run_at": "document_start",
-      "matches": ["http://*/*", "https://*/*"],
-      "js": ["scripts/blacklister.js"]
+      "matches": [
+        "file://*/*",
```

Comment:
I copied our other contentscript when I was trying to get it working. Removed this line since it doesn't really make sense.